spark-ui-proxy.py

```diff
@@ -86,6 +86,8 @@ def extractUrlDetails(self, path):
     def rewriteLinks(self, page, targetHost):
         target = "{0}proxy:{1}/".format(URL_PREFIX, targetHost)
         page = page.replace('href="/', 'href="' + target)
+        page = page.replace("'<div><a href=' + logUrl + '>'",
```

Comment:
`logUrl` doesn't seem to be a defined variable. I may be missing something here.

Reply:
`logUrl` is part of the string to match - it is in a javascript file loaded by the spark webui on the page that display the executors of an application.

Reply:
Right! I missed the quotes.